fe/fe-core/src/test/java/org/apache/doris/rewrite/RewriteBinaryPredicatesRuleTest.java

```diff
@@ -0,0 +1,75 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+package org.apache.doris.rewrite;
+
+import com.clearspring.analytics.util.Lists;
+import org.apache.doris.analysis.*;
+import org.apache.doris.common.FeConstants;
+import org.apache.doris.common.jmockit.Deencapsulation;
+import org.apache.doris.qe.ConnectContext;
+import org.apache.doris.utframe.DorisAssert;
+import org.apache.doris.utframe.UtFrameUtils;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.UUID;
+
+public class RewriteBinaryPredicatesRuleTest {
+    private static String baseDir = "fe";
+    private static String runningDir = baseDir + "/mocked/RewriteBinaryPredicatesRuleTest/"
+            + UUID.randomUUID() + "/";
+    private static DorisAssert dorisAssert;
+    private static final String DB_NAME = "rewritedaterule";
+    private static final String TABLE_NAME_1 = "tb1";
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        FeConstants.default_scheduler_interval_millisecond = 10;
+        FeConstants.runningUnitTest = true;
+        UtFrameUtils.createDorisCluster(runningDir);
```

Comment:
call `UtFrameUtils.createDorisCluster` will increase the running time of unit test.
You can refer to `RewriteDateLiteralRuleTest.java`

Reply:
ok, i will recommit the test case